### ISSUE TITLE
[ui] Additional Logging Configuration update

### ISSUE THREAD

**Issue by dramich** [rancher/rancher#24367]
An option for JSON parsing needs to be added under the Additional Logging Configuration section when enabling a logging provider. 

A new field `enableJSONParsing` is being added to the logging config. 

Related issue: https://github.com/rancher/rancher/issues/23646
backend PR: https://github.com/rancher/rancher/pull/24361

**Comment by westlywright** [comment]
Available in `latest2` and `latest-2.3`

**Comment by izaac** [comment]
@westlywright it seems we're not sending true in the UI for the toplevel `enableJSONParsing` ? 

https://github.com/rancher/rancher/issues/23646#issuecomment-564723871

**Comment by dramich** [comment]
The `enableJSONParsing ` only lives on `clusterLogging`, not the logging provider configs. It's global for everybody.

**Comment by izaac** [comment]
Validated this is working at the project level in versions:

```
2.4 (12/16/2019) Rancher version bbb75e098
2.3-head (12/16/2019) Rancher version 78ed261c6 
```

Cluster level logging I hit the following issue, which blocks me to finish the validation. https://github.com/rancher/rancher/issues/24545 

@westlywright @dramich @sangeethah

**Comment by izaac** [comment]
validated working after fix of #24545 

```
2.3-head (12/27/2019)
2019/12/27 16:05:06 [INFO] Rancher version e9a549e59 is starting

master-head (12/27/2019)
2019/12/27 16:33:53 [INFO] Rancher version 189e4337c is starting
```
